jdaviz/configs/default/plugins/subset_plugin/subset_plugin.py

```diff
@@ -199,6 +200,11 @@ def _unpack_get_subsets_for_ui(self):
                 self.glue_state_types = self.glue_state_types + [glue_state]
                 self.subset_states = self.subset_states + [subset_state]
 
+        if len(self.subset_states) > 1 and isinstance(self.subset_states[0], RangeSubsetState):
```

Comment:
technically the internal logic is more restrictive and actually requires a _spectral_ subset... but I'm wondering now if instead `get_subsets(spectral_only=True)` could be `get_subsets(range_only=True)` and then `simplify_spectral_subset` could be renamed `simplify_range_subset`?

I know that might be out of scope for this PR, but otherwise this lack of generality is going to result in bugs that will need to be fixed later here or hacked around in lcviz 🤔 .  As this is right now, lcviz shows the simplify button because they are range subsets, but then clicking the button raises a traceback because the range subsets aren't found with `spectral_only = True`.

Reply:
We talked offline and I think these are great points to bring up with the team, namely how do we handle jdaviz being an upstream package. For now, I think this is out of scope for this PR but all points are noted for future follow-up work.

Reply:
after offline discussion, let's ignore this for now and revisit as a follow-up.  This PR isn't currently public API, so we can iterate as needed to generalize for possible downstream applications later.